Automattic/team51-cli
# Feature/install safety net on dev sites

## Synopsis

Adds the Safety Net plugin to every new development site created, unless the `skip-safety-net` option is used.

## To Test

- Create a development site and verify Safety Net is loaded as an mu-plugin. e.g. `team51 create-development-site --site-id=1200878 --temporary-clone --label=nickg-safety-net-test`
- Create a development site, and skip adding Safety Net. e.g. `team51 create-development-site --site-id=1200878 --temporary-clone --label=nickg-safety-net-test --skip-safety-net`
